frontends/web/src/components/amount/amount.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import style from './amount.module.css';
+
+type TProps = {
+  amount: string;
+  unit: string;
```

Comment:
can't we use the account.ts:`CoinWithSat` type here?

would also be nice to rename `Coin` to `CoinUnit` and `CoinWithSat` to `CoinUnitWithSat` - was pretty confused about these type names when I looked them up now.

Reply:
`Amount` in this PR is used for both fiat and coin amounts, so `unit` should be something like `ConversionUnit | CoinWithSAT`. What about just renaming `CoinWithSAT` to `CoinUnit`? I think it would be nice to separate coins from their units and that would mean having `unit: ConversionUnit | CoinUnit` which can be easier to understand. wdyt?

Reply:
makes sense, and the `Coin/MainnetCoin/TestnetCoin` ones can actually be inlined and deleted. Does this commit make sense?

https://github.com/benma/bitbox-wallet-app/commit/c839829c2e2a83d5910eb6205a7f4106056fdba5

Reply:
Yeah, you're right! I included your modifications in my commit, thanks